lint/README.md

```diff
@@ -0,0 +1,43 @@
+# GitHub Action Lint
+
+## Description
+
+GitHub Action that lints a Node based repository
+
+## Usage
+
+```yaml
+jobs:
+  build:
+    steps:
+      - name: Lint
+        uses: open-turo/actions-node/lint@v1
+        with:
+          ## example value for github-token provided below
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+## Inputs
+
+| parameter      | description                                                                | required | default |
+| -------------- | -------------------------------------------------------------------------- | -------- | ------- |
+| checkout-repo  | Perform checkout as first step of action                                   | `false`  | true    |
+| eslint-flags   | Flags and args of eslint command                                           | `false`  |         |
+| github-token   | GitHub token that can checkout the repository. e.g. 'secrets.GITHUB_TOKEN' | `true`   |         |
+| npm-auth-token | The Node Package Manager (npm) authentication token                        | `false`  |         |
```

Comment:
These docs are generated from the `action.yaml` using https://github.com/npalm/action-docs - and manually integrated into the README. 
```suggestion
| parameter      | description                                                                | required | default |
| -------------- | -------------------------------------------------------------------------- | -------- | ------- |
| checkout-repo  | Perform checkout as first step of action                                   | `false`  | true    |
| github-token   | GitHub token that can checkout the repository. e.g. 'secrets.GITHUB_TOKEN' | `true`   |         |
```